src/test/java/harness/utils/GcpDataprocClusterUtils.java

```diff
@@ -0,0 +1,133 @@
+package harness.utils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import bio.terra.cli.serialization.userfacing.resource.UFGcpDataprocCluster;
+import bio.terra.cli.service.utils.HttpUtils;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import harness.TestCommand;
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GcpDataprocClusterUtils {
+  /**
+   * Helper method to poll `terra resources describe` until the cluster state equals that specified.
+   * Uses the current workspace.
+   */
+  public static void pollDescribeForClusterState(String resourceName, String clusterState)
+      throws InterruptedException, JsonProcessingException {
```

Comment:
Why not use ResourceUtils.pollDescribeForResourceField()  ? The timeout can be made into a param if needed. 
Its pretty much the same functionality